pkg/costattribution/manager.go

```diff
@@ -21,6 +21,7 @@ const (
 	defaultTrackerName = "cost-attribution"
 	missingValue       = "__missing__"
 	overflowValue      = "__overflow__"
+	usagePrefix        = "attributed"
```

Comment:
Nit, but I would expect the prefix to be `attributed_`.

Reply:
addressed in 97a5d77efb0c7b63dd34d6fa8b5d2216d7334373